hbase-handler/src/test/org/apache/hadoop/hive/hbase/TestHBaseStorageHandler.java

```diff
@@ -46,6 +53,46 @@ public void testHbaseConfigIsAddedToJobConf() {
         jobConfToConfigure.get("hbase.some.fake.option.from.xml.file") != null);
   }
 
+  @Test
+  public void testGetUriForAuth() {
```

Comment:
The method seems to contains five tests. They all test `getURIForAuth` method but the input for every test is different. I am a big fan of keeping one assert per test method which tends to make test more readable and maintainable. There are various books (e.g., "The art of unit testing" by Roy Osherove) which go into more detail of why this is a good idea and in this case it seems beneficial to split and refactor the method into five separate tests. 
The common test code could go into a method `checkUriForAuth(serdeParams, expectedUri)` and the 5 test methods would call this with a different set of parameters.